docs/NooBaaNonContainerized/Upgrade.md

```diff
@@ -6,7 +6,11 @@
 4. [Offline Upgrade (Version < 5.18.0)](#offline-upgrade-version--5180)
     1. [Offline Upgrade steps](#offline-upgrade-steps)
 5. [Online Upgrade (Version >= 5.18.0)](#online-upgrade-version--5180)
-
+    1. [Online Upgrade Goals](#online-upgrade-goals)
+    2. [Online Upgrade Algorithm](#online-upgrade-algorithm)
+    3. [Additional Upgrade Properties of `system.json`](#additional-upgrade-properties-of-systemjson)
```

Comment:
I could click on it and get to the section.
(if you want to try: tree dots on the right header if the file in GH -> view file).

Reply:
thanks, for some reason it works on VSC and not in here